core/src/main/java/com/es/core/model/phone/JdbcPhoneDao.java

```diff
@@ -13,15 +13,33 @@
     @Resource
     private JdbcTemplate jdbcTemplate;
 
+    public void setJdbcTemplate(JdbcTemplate jdbcTemplate) {
```

Comment:
It's not needed if resource is used, it's used when injected bean is registered through property in xml config